include/drivers/hwinfo.h

```diff
@@ -30,6 +30,18 @@
 extern "C" {
 #endif
 
+enum reset_cause {
+	RESET_PIN				= BIT(0),
+	RESET_SOFTWARE				= BIT(1),
+	RESET_BROWNOUT				= BIT(2),
+	RESET_POWER				= BIT(3),
```

Comment:
Should this be `RESET_POWER_ON`, especially noting that there are other "power"-related events such as `RESET_BROWNOUT` and `RESET_LOW_POWER_WAKE`.

Reply:
Probably better to go with `RESET_POR`